src/cli/cli.ts

```diff
@@ -1,3 +1,4 @@
+import * as Path from "path";
```

Comment:
Question: Any reason to not just import the `join` method here?

Reply:
I like to keep standard library functions namespaced